### DISCUSSION TITLE
Can someone kindly help with this error message below

### DISCUSSION THREAD

**Discussion by MicroFarmCTO** [jup-ag/token-list#1436] [Q&A]
Run peter-evans/create-pull-request@v5
Error: Input 'token' not supplied. Unable to continue.
Restore git configuration
  Error: Cannot read properties of undefined (reading 'removeAuth')

**Comment by MONTPBM** [comment]
WTH IS GOING ON WITH MY MONEY AND TOKENS?! This seems very SCAMMY! I WANT ALL OF MY MONEY WHEN IT SHOT ALL THE WAY UP AND FROZE US OUT FROM SELLING! NOW!
